src/KrakenBattle.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * File:   KrakenBattle.cpp
+ * Author: Kjell Hedstrom
+ *
+ * Created on November 24, 2015
+ */
+
+#include "KrakenBattle.h"
+#include <algorithm>
+#include <iterator>
+#include <g3log/g3log.hpp>
+
+namespace {
+   const std::string emptyUUID = {"00000000-0000-0000-0000-000000000000"};
+   const std::vector<uint8_t> emptyData = {};
+}
+
+
+
+namespace KrakenBattle {
+
+   /**
+   * formats the data to send to the Harpoon according to the specification for
+   *  @ref file and pcap sending
```

Comment:
@john-gress  what would you like to be specified for the Kraken/Harpoon message schema?